### PR TITLE
Add intel-ipp recipe

### PR DESCRIPTION
**Pull request by willprice**
Intel IPP is a dependency of [accimage](https://github.com/conda-forge/staged-recipes/pull/7610) which I've built a recipe for.

### CODE REVIEW

**Comment by willprice** [comment]
@conda-forge/staged-recipes
Ready for review :tada:

File: recipes/intel-ipp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  url: http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14887/l_ipp_{{ version }}.tgz
+  sha256: 1eb7cd0fba74615aeafa4e314c645414497eb73f1705200c524fe78f00620db3
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (scopatz) [inline recipes/intel-ipp/meta.yaml]: Seems like you actually need python here, not a c compiler

Comment (willprice) [reply recipes/intel-ipp/meta.yaml]: Good catch, thanks

**Comment by willprice** [comment]
@conda-forge/staged-recipes @scopatz would it be possible to review this in the next few days? Thanks :)

**Comment by scopatz** [comment]
LGTM Thanks!

**Comment by willprice** [comment]
Thanks @scopatz much appreciated!

**Comment by scopatz** [comment]
No problem! Feel free to ping whenever it is ready in the future!

**Pull request merged**